spec/rubocop/cop/rspec/expect_actual_spec.rb

```diff
@@ -135,6 +207,108 @@
     RUBY
   end
 
+  it 'flags `be` with no argument' do
+    expect_offense(<<~RUBY)
+      describe Foo do
+        it 'uses expect incorrectly' do
+          expect(1).to be
+                 ^ Provide the actual you are testing to `expect(...)`.
```

Comment:
Well, we don't have an actual here, and supporting this case complicates the pattern and the autocorrect code

Reply:
Makes sense.

Reply:
Fixed.